eth/watchers/orchestratorwatcher.go

```diff
@@ -186,26 +184,34 @@ func (ow *OrchestratorWatcher) handleRoundEvent(log types.Log) error {
 	}
 
 	for _, o := range orchs {
-		if err := ow.cacheOrchestratorStake(ethcommon.HexToAddress(o.EthereumAddr), round); err != nil {
-			glog.Errorf("could not cache stake update for orchestrator %v and round %v", o.EthereumAddr, round)
-		}
+		go ow.cacheOrchestratorStake(ethcommon.HexToAddress(o.EthereumAddr), round)
 	}
 
 	return nil
 }
 
 func (ow *OrchestratorWatcher) cacheOrchestratorStake(addr ethcommon.Address, round *big.Int) error {
-	ep, err := ow.lpEth.GetTranscoderEarningsPoolForRound(addr, round)
+	var err error
+
+	defer func() {
```

Comment:
I think you can improve on this error handling:
1. You don't need to return an error from this function if it's not used.

2. I'd avoid using `defer func() {}` just for logging the error. Especially that depending on the reason for an error you could log something different. For example,

```
	ep, err := ow.lpEth.GetTranscoderEarningsPoolForRound(addr, round)
	if err != nil {
		glog.Errorf("could not cache stake update for orchestrator %v and round %v, getting transcoder earnings pool for round failed err=%q", o.EthereumAddr, round, err)
        return
	}
```

Reply:
OK, will fully run the node and observe the output. The issue body will be updated once done.